src/Compilers/CSharp/Test/Semantic/Semantics/SlowTests.cs

```diff
@@ -10,7 +10,8 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Semantics
 {
     public class SlowTests : CSharpTestBase
     {
-        [Fact, WorkItem(35949, "https://github.com/dotnet/roslyn/issues/35949")]
+        [WorkItem(35949, "https://github.com/dotnet/roslyn/issues/35949")]
+        [ConditionalFact(typeof(IsRelease))]
         public void NotNull_Complexity()
```

Comment:
@jcouv, can we remove this test class? It looks identical to the test in OverloadResolutionPerfTests.cs.

Reply:
Good catch. Sure let's remove this one (and thus the file). 
Not sure how this god introduced, I just moved this test out of NullableReferenceTypeTests since it was slow...

Reply:
Done